src/utils/registration-helpers.ts

```diff
@@ -818,30 +841,40 @@ export const convertToRegistrationSearchItem = (registration: Registration) => {
       ? registration.entityDescription.reference.publicationContext.series
       : undefined;
 
+  const contributors = registration.entityDescription?.contributors.map((contributor) => ({
+    affiliation: contributor.affiliations,
+    correspondingAuthor: contributor.correspondingAuthor,
+    identity: contributor.identity,
+    role: contributor.role.type,
+  }));
+
   const registrationSearchItem: RegistrationSearchItem = {
+    type: registration.entityDescription?.reference?.publicationInstance.type ?? '',
     id: registration.id,
     identifier: registration.identifier,
-    createdDate: registration.createdDate,
-    modifiedDate: registration.modifiedDate,
-    publishedDate: registration.publishedDate,
-    status: registration.status,
-    entityDescription: {
-      mainTitle: registration.entityDescription?.mainTitle ?? '',
-      abstract: registration.entityDescription?.abstract ?? '',
-      description: registration.entityDescription?.description ?? '',
-      publicationDate: registration.entityDescription?.publicationDate,
-      contributorsPreview: registration.entityDescription?.contributors ?? [],
-      contributorsCount: (registration.entityDescription?.contributors ?? []).length,
-      reference: {
-        publicationInstance: {
-          type: registration.entityDescription?.reference?.publicationInstance?.type,
-        },
-        publicationContext: {
-          publisher,
-          series,
-        },
-      },
+    recordMetadata: {
+      createdDate: registration.createdDate,
+      modifiedDate: registration.modifiedDate,
+      publishedDate: registration.publishedDate,
+      status: registration.status,
     },
+    mainTitle: registration.entityDescription?.mainTitle ?? '',
+    contributorsCount: (registration.entityDescription?.contributors ?? []).length,
+    abstract: registration.entityDescription?.abstract ?? '',
+    description: registration.entityDescription?.description ?? '',
+    publicationDate: registration.entityDescription?.publicationDate,
+    publishingDetails: publisher
```

Comment:
Her blir det kanskje feil om det er tidsskrift som ligger her og ikke utgiver?
Ser vi hadde lignende før også da, så usikker på akkurat hva som skjer her 🤔 får grave litt mer

Reply:
Kan se litt rart ut å falbacke til `emptyContextPublisher` også, siden vi da vil ende opp med tom beskrivelse av publisher om det er journal som ligger der.. Ikke testet så mye enda, men tror noe sånt vil være veien å gå:

```
const publishingDetailsId = registration.entityDescription?.reference?.publicationContext
  ? 'publisher' in registration.entityDescription.reference.publicationContext
    ? registration.entityDescription.reference.publicationContext.publisher?.id
    : 'id' in registration.entityDescription.reference.publicationContext
      ? (registration.entityDescription.reference.publicationContext.id ?? undefined)
      : undefined
  : undefined;

(...)

publishingDetails: {
  type: registration.entityDescription?.reference?.publicationContext.type, // Sliter med å få denne til å gå opp atm
  id: publishingDetailsId,
  series: {
    name: series?.title,
    id: series?.id,
  },
  doi: registration.entityDescription?.reference?.doi,
}

```

Reply:
Okei, har noe greier her som ser ut til å virke.. Men syns det ble litt komplekst og aner ikke om det dekker alle caser 😬 
```
const basePublishingDetails = registration.entityDescription?.reference?.publicationContext
  ? 'publisher' in registration.entityDescription.reference.publicationContext
    ? {
        id: registration.entityDescription.reference.publicationContext.publisher?.id ?? '',
        type: PublicationChannelType.Publisher,
      }
    : 'id' in registration.entityDescription.reference.publicationContext
      ? {
          id: registration.entityDescription.reference.publicationContext.id ?? undefined,
          type: PublicationChannelType.Journal,
        }
      : {}
  : {};
(...)
publishingDetails: {
  ...basePublishingDetails,
  series: {
    name: series?.title,
    id: series?.id,
  },
  doi: registration.entityDescription?.reference?.doi,
},
```

Reply:
Burde ikke trenge alt dette nå lenger, når Sondre har lagt inn ny property for `publisher`(?). Tror det da vil være ganske likt som det var, bare med mulig tillegg for å sette tidsskrift